src/student/mainService/Service.java

```diff
@@ -0,0 +1,70 @@
+package student.mainService;
+
+import student.customImplementations.DoubleLinked;
+import student.customImplementations.DynamicArray;
+import student.models.Student;
+
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.Scanner;
+
+public class Service {
```

Comment:
create class names in the way that will be understandable what class responsible to do